randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -506,20 +506,18 @@ Extra - room_id: [13]
 > Beside Pickup; Heals? False
   * Layers: default
   > Pickup (Hidden Missile Tank)
```

Comment:
I would prefer if this item were called "Top Missile Tank" and the other "Bottom Missile Tank"